superset/assets/src/utils/common.js

```diff
@@ -108,3 +108,18 @@ export function prepareCopyToClipboardTabularData(data) {
   }
   return result;
 }
+
+
+// More functional switch cases
+// https://codeburst.io/alternative-to-javascripts-switch-statement-with-a-functional-twist-3f572787ba1c
+export const matched = x => ({
```

Comment:
I checked whether the snippets have a license and they don't seem to. I think it's so short that it's ok. We just need to build the reflex of checking license when copy/pasting snippets.